src/bin/cargo/commands/test.rs

```diff
@@ -41,9 +41,9 @@ pub fn cli() -> Command {
             "Test only the specified example",
             "Test all examples",
             "Test only the specified test target",
-            "Test all test targets",
+            "Test all targets that have the `test = true` manifest flag set",
```

Comment:
Could we make them shorter, like

```suggestion
            "Test all targets that have `test = true` set",
```

and ditto `--benches`?

(I still find it confusing though, because it doesn't really test "all" targets. Anyway this PR is already an improvement)